test/import-test.js

```diff
@@ -706,6 +709,7 @@ describe('#importDataElement', () => {
   it('Import49: should correctly import TBD concept', () => {
     const specifications = importFixture('ConceptTBD');
     const entry = expectAndGetEntry(specifications, 'conceptTBDOut', 'ConceptTBD');
+    console.log(entry);
```

Comment:
Did you mean to leave this `console.log` as-is (_not_ commented out)?

Reply:
that one should be removed.